board/utils.h

```diff
@@ -22,6 +22,11 @@
   (_a > 0) ? _a : (-_a); \
 })
 
+#define ROUND(a) ({ \
+  __typeof__ (a) _a = (a); \
+  _a + ((_a > 0) ? 0.5 : -0.5); \
```

Comment:
this just does the addition and results in a float. no need for a macro here either

Reply:
You mean you don't see a need for the macro, or the typeof operator is not needed?